src/main/java/gov/cms/madie/models/cqm/datacriteria/basetypes/DataElement.java

```diff
@@ -138,4 +144,49 @@ public ZonedDateTime shiftDateByYear(ZonedDateTime dateTime, int year) {
     }
     return shiftedDateTime;
   }
+
+  public Object shiftDateByYearForObject(Object object, int year) {
+    if (object == null) {
+      return null;
+    }
+
+    Object convertedObj = object;
+    // Date type
+    try {
+      LocalDate converted = convertToLocalDate(object);
+      if (converted != null) {
+        converted = converted.plusYears(year);
```

Comment:
Need to limit the resulting year range to [1900, 9999] to avoid display issues.

Reply:
@jkotanchik-SB Thanks so much Joe! I added the limitation.